src/Umbraco.Cms.Integrations.Crm.Dynamics/Umbraco.Cms.Integrations.Crm.Dynamics.csproj

```diff
@@ -23,9 +23,21 @@
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net50'">
-		<PackageReference Include="Umbraco.Cms.Web.Website" version="9.0.1" />
-		<PackageReference Include="Umbraco.Cms.Web.BackOffice" version="9.0.1" />
-		<PackageReference Include="Umbraco.Cms.Web.Common" version="9.0.1" />
+		<PackageReference Include="Umbraco.Cms.Web.Website" version="[9.0.1,10)" />
+		<PackageReference Include="Umbraco.Cms.Web.BackOffice" version="[9.0.1,10)" />
+		<PackageReference Include="Umbraco.Cms.Web.Common" version="[9.0.1,10)" />
+	</ItemGroup>
+
+	<ItemGroup Condition="'$(TargetFramework)' == 'net60'">
+		<PackageReference Include="Umbraco.Cms.Web.Website" version="[10.0.0,11)" />
+		<PackageReference Include="Umbraco.Cms.Web.BackOffice" version="[10.0.0,11)" />
+		<PackageReference Include="Umbraco.Cms.Web.Common" version="[10.0.0,11)" />
+	</ItemGroup>
+
+	<ItemGroup Condition="'$(TargetFramework)' == 'net70'">
+		<PackageReference Include="Umbraco.Cms.Web.Website" version="[11.0.0,12]" />
```

Comment:
@acoumb I think this is a mistake, it should have a round brace behind v12, instead of a straight bracket. 
I have a report from a user that can't upgrade to v12 because of this.  I've tested it as well and I get the error below.
Must have worked in your tests as it was a pre-release, so maybe that doesn't count as v12 yet? 😅 

![image](https://github.com/umbraco/Umbraco.Cms.Integrations/assets/304656/d7af43ad-2a03-4c08-83d3-8128f1ec5de2)

Reply:
This has been addressed with a new patch release. 
Thank you